source/docs/quick_start.md

```diff
@@ -1,61 +1,38 @@
 title: Quick Start
 ---
 
-### Installation
+In this guide we'll explore how easy it is to quickly get up and running with Embark to build our first decentralized application. We assume you've been through our [installation guide](installation.html) and have everything you need to get started. If not, please head over there first and get your machine ready.
 
-First make sure you have installed Embark, preferably using NVM. See [install instructions](index.html#Requirements)
+### Creating your first app
 
-### Create demo app
+In order to get results as quickly as possible, Embark comes with a `demo` command that scaffolds and sets up a complete application for us to play with. Open up a terminal of your choice and run:
 
 <pre><code class="shell">$ embark demo
-$ cd embark_demo
+$ cd embark demo
 </code></pre>
 
-### Start a blockchain node or let Embark start it for you
+This will create a demo application. Feel free to look around to get familiar with the project's structure, but don't worry, we'll take a closer look at it later in one of our [dedicated guides](structure.html).
```

Comment:
`but don't worry, we'll take a closer look` -> `but don't worry. We'll take a closer look`

Reply:
Updated!